spec/features/web_intake/requested_documents_token_spec.rb

```diff
@@ -45,4 +45,35 @@
     visit "/questions/job-count"
     expect(current_path).to eq(welcome_questions_path())
   end
+
+  context "with forgery protection" do
+    around do |example|
+      ActionController::Base.allow_forgery_protection = true
+      Capybara.raise_server_errors = false
+      example.run
+      Capybara.raise_server_errors = true
+      ActionController::Base.allow_forgery_protection = false
+    end
+
+    scenario "client completes requested docs and gets message when trying to go back", :js do
```

Comment:
We should be able to test this without Javascript. Is the intention here to test the javascript functionality?

Reply:
I tried to make these tests work without JS for about an hour and made no progress. We'll try to catch you for an in-person conversation but I don't think this needs to block release of the bugfix.